runtime/service/src/main/java/org/apache/polaris/service/task/TaskExecutorImpl.java

```diff
@@ -52,18 +62,28 @@ public class TaskExecutorImpl implements TaskExecutor {
   private final TaskFileIOSupplier fileIOSupplier;
   private final List<TaskHandler> taskHandlers = new CopyOnWriteArrayList<>();
   private final PolarisEventListener polarisEventListener;
+  private final Tracer tracer;
```

Comment:
I'd prefer to make this a 
```suggestion
  private final Optional<Tracer> tracer;
```

I'm a bit cautious when using otel code, been bitten hard w/ static initializers in the past, although the particular test doesn't seem to harm.